purchase_split_wizard/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 {
     "name": "Purchase Split Wizard",
     "summary": "Wizard to split purchase orders into multiple ones",
-    "version": "16.0.1.0.0",
+    "version": "16.0.1.0.1",
```

Comment:
you should not change the version as the bot does it, undo the change.

Reply:
@ValentinVinagre done